app.rb

```diff
@@ -0,0 +1,146 @@
+require_relative 'person'
+require_relative 'classroom'
+require_relative 'book'
+require_relative 'rental'
+require_relative 'student'
+require_relative 'teacher'
+def list_all_books(books)
+  puts 'List of books:'
+  puts
+  books.each_with_index do |book, index|
+    puts "#{index + 1}. Title: #{book.title}, Author: #{book.author}"
+    puts
+  end
+end
+
+def list_all_people(people)
+  puts 'List of people:'
+  puts
+  people.each_with_index do |person, index|
+    person_type = person.instance_of?(Student) ? '[Student]' : '[Teacher]'
+    puts "#{index + 1}. #{person_type} Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
+    puts
+  end
+end
```

Comment:
- [ ]  Hi, great job for applying the changes requested by the previous review. In running `ruby main.rb`, I noticed that the changes you made did not make the App to work as expected. Can you try to fix the issues, and check if everything is working? Because the App must work as expected.
- NB: Pay attention to the red line in the picture to understand what I am talking about
![Capture](https://github.com/Adholah96/school-library/assets/63972670/acde04ca-de27-461f-a6e8-8233c610dd2c)